tests/mypy_test.py

```diff
@@ -573,8 +575,8 @@ def main() -> None:
             total_packages_skipped += packages_skipped_this_version
     if code:
         plural = "" if total_files_checked == 1 else "s"
-        print_error(f"--- exit status {code}, {total_files_checked} file{plural} checked ---")
-        sys.exit(code)
+        print_error(f"--- mypy exit status {code}, {total_files_checked} file{plural} checked ---")
+        sys.exit(1)
```

Comment:
I don't think it makes much sense to propagate just one exit code to the end, since mypy might fail multiple times. I'll just add an enum to signal the "worst" exit code.

Reply:
Yeah, agreed that the existing behaviour didn't really make much sense! I like the change you made to print the returncode for each mypy failure.